src/components/Text/Heading/index.tsx

```diff
@@ -4,36 +4,39 @@ import classNames from 'classnames';
 import { useScreenSize } from '../../../hooks/useScreenSize';
```

Comment:
My bad - changes only from prettier after reverting one of my changes. No actual changes to this file, can be ignored on conflict